v2-apache/Dockerfile

```diff
@@ -183,4 +184,6 @@ RUN set -eux; \
     chgrp -R 0 /var/log/ /usr/local/apache2/; \
     chmod -R g=u /var/log/ /usr/local/apache2/
 
+HEALTHCHECK CMD curl -sk https://${SERVER_NAME}:${SSL_PORT}/healthz
```

Comment:
- protocol should probably depend on `SSL_ENGINE`, same as port.
- shouldn't the host name always be localhost? Otherwise the request might go out through the network before returning back to the container... (not sure)

Reply:
Changed the one-liner to a script so we can deal with both servers at once.
- SSL_ENGINE is set only for apache
- moved to localhost. Will not be a problem either case I guess.